hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -231,7 +234,8 @@ protected void onExpire(K key, R record) {
     }
 
     protected boolean isEvictionEnabled() {
-        return evictionStrategy != null && evictionPolicyEvaluator != null;
+        return evictionStrategy != null && evictionPolicyEvaluator != null
```

Comment:
To improve the readability of such expressions I often do something like 

```
return evictionStrategy != null
     && evictionPolicyEvaluator != null
     && evictionPolicyType != null
     && !evictionPolicyType.equals(EvictionPolicyType.NONE)
```
Then it becomes easier to see the structure